src/app/shared/components/ds-forms/shortdate-picker/shortdate-picker.component.html

```diff
@@ -19,7 +19,7 @@
     [class.with-item-left]="icon"
     [disabled]="checkDisabled()"
     bsDatepicker
-    placeholder="Select date"
+    placeholder="Select date range"
```

Comment:
Remove `range` here.  This section of code renders a non-range datepicker.

Reply:
The other one on line 38 is okay?

Reply:
Yessir